cargo-flash/src/main.rs

```diff
@@ -16,33 +16,33 @@ use probe_rs_cli_util::logging::{ask_to_log_crash, capture_panic};
 
 use probe_rs_cli_util::{build_artifact, log, logging, logging::Metadata};
 
-lazy_static::lazy_static! {
-    static ref METADATA: Arc<Mutex<Metadata>> = Arc::new(Mutex::new(Metadata {
+fn main() {
+    let metadata: Arc<Mutex<Metadata>> = Arc::new(Mutex::new(Metadata {
         release: meta::CARGO_VERSION.to_string(),
         chip: None,
         probe: None,
         speed: None,
         commit: git_version::git_version!(fallback = "crates.io").to_string(),
     }));
-}
 
-fn main() {
+    let metadata_panic = metadata.clone();
+
     let next = panic::take_hook();
     panic::set_hook(Box::new(move |info| {
         #[cfg(feature = "sentry")]
         if ask_to_log_crash() {
-            capture_panic(&METADATA.lock().unwrap(), info)
+            capture_panic(&metadata_panic.lock().unwrap(), info)
         }
         #[cfg(not(feature = "sentry"))]
-        log::info!("{:#?}", &METADATA.lock().unwrap());
+        log::info!("{:#?}", &metadata_panic.lock().unwrap());
         next(info);
     }));
 
-    match main_try() {
+    match main_try(metadata.clone()) {
```

Comment:
This clone can be removed :)

Reply:
This is trickier than it seems, with `--all-features` the clone is not necessary, but without it was.